### PR TITLE
Release/2019 04

### PR DESCRIPTION
**Pull request by gallayl**

### CODE REVIEW

File: apps/sensenet/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+## 🔑 Login, change repository
+
+To manage a repository, you have to login in first. You have to provide
+
+- Your username (e.g. _my-domain\\my-user_ or _myprofile@myorganization.com_)
+- The password for the user
+- The repository URL that you want to manage (e.g. _https://my-sensenet-installation.com_)
+  Once you've logged in, the repository and the last username will be saved into your _personal settings_.
+
+If you want to use an another repository, you can switch with the repository selector on the top left corner. (or login to a new repository with the **Another repository** link)
```
Comment (kultsar) [inline apps/sensenet/README.md]:
```suggestion
If you want to use another repository, you can switch with the repository selector on the top left corner (or login to a new repository with the **Another repository** link).
```

File: apps/sensenet/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+Your repository sessions will be managed individually, you can use multiple repositories simultaneously. The repository selector will also indicates the current session state.
```
Comment (kultsar) [inline apps/sensenet/README.md]:
```suggestion
Your repository sessions will be managed individually, you can use multiple repositories simultaneously. The repository selector will also indicate the current session state.
```

File: apps/sensenet/README.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+The repositories you've visited will be also saved in your Personal Settings - you can add a meaningful _"displayName"_ to them here
+
+## 🌍 Content
+
+You can browse the whole repository with the **Content** menu.
+You can adjust the Content view in the personal setting's _"content"_ section.
+
+## 🌈 Command palette
+
+The command palette is useful is you want to search in the repository, navigate to a specific page or execute a specific command on the current content.
```
Comment (kultsar) [inline apps/sensenet/README.md]:
```suggestion
The command palette is useful if you want to search in the repository, navigate to a specific page or execute a specific command on the current content.
```

File: apps/sensenet/README.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+You can activate the command palette from the top right **">\_"** icon - or the **ctrl-p** shortcut.
```
Comment (kultsar) [inline apps/sensenet/README.md]:
```suggestion
You can activate the command palette from the top right **">\_"** icon - or with the **ctrl-p** shortcut.
```

File: apps/sensenet/README.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+### Search by path
+
+You can search content by its path if you start typing with "/Root", you will get completition with one level below
```
Comment (kultsar) [inline apps/sensenet/README.md]:
```suggestion
You can search content by its path if you start typing with "/Root", you will get completition with one level below.
```

File: apps/sensenet/src/services/CommandProviders/CheatCommandProvider.ts [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+        Id: 0,
+        Path: '',
+        Name: '',
+        Type: 'DoomCheatContent',
+      },
+    },
+    'pot of gold': {
+      primaryText: 'cheat enabled you wascally wabbit',
+      secondaryText: 'now take a screenshot and go talk to your line manager',
+      url: 'https://www.myabandonware.com/game/warcraft-orcs-humans-250/play-250',
```
Comment (kultsar) [inline apps/sensenet/src/services/CommandProviders/CheatCommandProvider.ts]: This returns a 404

Comment (gallayl) [reply apps/sensenet/src/services/CommandProviders/CheatCommandProvider.ts]: it was just released on [GoG](https://www.gog.com/game/warcraft_bundle) - but this URL is unused at the moment. We should create a component for old DOS games in the future :)

File: package.json [hunk 7]
```diff
@@ -1,10 +1,11 @@
 {
   "private": true,
-  "version": "2019-03",
+  "version": "2019-04",
```
Comment (B3zo0) [inline package.json]: while we don't publish this to npm I think that we should stick with the [semver](https://docs.npmjs.com/misc/semver) syntax for the version field. eg.: 2019.4.1 or 2019.4.0

**Pull request closed without merging**